client/src/unifycr.c

```diff
@@ -2643,7 +2647,10 @@ int unifycrfs_mount(const char prefix[], size_t size, int rank)
     }
 
     /* initialize our library */
-    unifycr_init(rank);
+    int ret = unifycr_init(rank);
+
+    if (ret != UNIFYCR_SUCCESS)
+        return UNIFYCR_FAILURE;
```

Comment:
Small nit here, but I'd suggest returning `ret` here. My thinking is that in the future `unificy_init()` might return different error codes depending on the failure type, and we'd want to pass that back to the application.

Reply:
@nedbass Thanks, good catch. I updated it.